jenkins/aws/manageS3Registry.sh

```diff
@@ -255,6 +262,26 @@ function copyToRegistry() {
         fatal "Unable to tag ${BASE_REGISTRY_FILENAME} as latest" && exit
 }
 
+# Remove the source S3 content. This is used to keep the S3 Stage clean for new uploads
+function removeSource() { 
+    local FILE_TO_REMOVE="${1}"
+
+    info "removing ${FILE_TO_REMOVE}"
+
+    if [[ "${FILE_TO_REMOVE}" =~ ^s3:// ]]; then
+        
+        aws --region "${REGISTRY_PROVIDER_REGION}" s3 ls "${FILE_TO_REMOVE}"  >/dev/null 2>&1
+        RESULT=$?
+        [[ "$RESULT" -ne 0 ]] &&
+            fatal "Can't access ${FILE_TO_REMOVE}" && exit
```

Comment:
exit or return 1? Its a function so I[d expect a return code that the caller could check before exitting

Reply:
Fixed